openstf/feature_engineering/general.py

```diff
@@ -76,3 +80,37 @@ def remove_extra_feature_columns(
         )
 
     return input_data.drop(extra_features, axis=1)
+
+
+def enforce_feature_order(df):
```

Comment:
I'm thinking about if `enforce_feature_order` is the right name for what this function does. Perhaps `sort_features` or `sort_feature_columns` is more descriptive of what it does? I guess it does enforce the order, but it does that by sorting.

Reply:
Yes but the first column ("load") and last column ('Horizon') are excluded from the sorting and therefore the end result is not a sorted dataframe. I think therefore the enforce order is more descriptive: we want to enforce this order: ["load"] -- [alphabetically sorted features] -- ['horizon']

Reply:
`sort_features_columns` also tells you that the function only sorts the features columns. In my opinion this implies that it does not sort non feature columns like the "horizon" and "load" column. But its not a really big deal.